containers/go-postgres/.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,61 @@
+version: '3.8'
+
+volumes:
+  postgres-data:
+
+services:
+  app:
+    container_name: godev
+    build: 
+      context: .
+      dockerfile: Dockerfile
+      args:
+        # Update 'VARIANT' to pick an version of Go: 1, 1.7.
+        # Append -bullseye or -buster to pin to an OS version.
+        # Use -bullseye variants on local arm64/Apple Silicon.
+        VARIANT: 1-bullseye
+        # Options
+        NODE_VERSION: "lts/*"
+    environment:
+      # NOTE: POSTGRES_DB/USER/PASSWORD should match values in devcontainer.json and in db container
+        POSTGRES_PASSWORD: ${POSTGRES_PASS}
+        POSTGRES_USER: ${POSTGRES_USER}
+        POSTGRES_DB: ${POSTGRES_DB}
+        POSTGRES_HOSTNAME: postgresdb
+    
+    # Security Opt and cap_add allow for C++ based debuggers to work. 
```

Comment:
Yep. I was also concerned, but the basic Go container has these enabled as well so that the dev container can run a debugger. Without these, the C++ debuggers for Go will fail.